closed/make/modules/ibm.healthcenter/Copy.gmk

```diff
@@ -48,8 +48,8 @@ TRANSPORT_PROPERTY_SED_SCRIPT := -e 's|$(TRANSPORT_PROPERTY_REGEX)\s*=.*|$(TRANS
 HEALTHCENTER_COPY := $(JDK_OUTPUTDIR)/modules/$(MODULE)/_the.$(MODULE)_copy_marker
 
 HEALTHCENTER_JARS := \
-	$(HEALTHCENTER_HOME)/healthcenter.jar \
 	$(HEALTHCENTER_HOME)/monitoring-api.jar \
+	$(HEALTHCENTER_HOME)/healthcenter.jar \
```

Comment:
Pls add a comment to the code to explain the ordering. Otherwise they may be reordered in the future to be alphabetical.

Reply:
Thanks Peter. Added comment, please review.